### PR TITLE
Regression tests: Bespoke Milestones

### PR DESCRIPTION
**Pull request by samdasari9**
Pull request created - Added Bespoke Milestone for Catalogue solution- 18/8/23

### CODE REVIEW

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/OrderingPages.cs [hunk 1]
```diff
@@ -295,8 +300,8 @@ public void StepThreeCompleteContract(bool isDefault = true)
 
             if (isOrderWithAssociatedService)
             {
-                TaskList.AssociatedServiceBillingAndRequirementsTask();
-                OrderingStepThree.SelectAssociatedServicesBilling(isDefault);
+            TaskList.AssociatedServiceBillingAndRequirementsTask();
```
Comment (ghost) [inline tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/OrderingPages.cs]: This file can be reverted as it seems the only change is removing the indentation

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepThree/OrderingStepThree.cs [hunk 2]
```diff
@@ -12,26 +18,18 @@ internal class OrderingStepThree : PageBase
     {
         public OrderingStepThree(IWebDriver driver, CommonActions commonActions)
             : base(driver, commonActions)
-        {
-        }
+            {
```
Comment (ghost) [inline tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepThree/OrderingStepThree.cs]: Indentation should be the same as the original

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepThree/OrderingStepThree.cs [hunk 3]
```diff
@@ -43,46 +41,45 @@ public void SelectAssociatedServicesBilling(bool isDefault = true)
         {
             if (isDefault)
             {
-                CommonActions.ClickFirstRadio();
-
-                CommonActions.ClickSave();
+              CommonActions.ClickSave();
```
Comment (ghost) [inline tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepThree/OrderingStepThree.cs]: Indentation is off by 2 spaces

File: src/NHSD.GPIT.BuyingCatalogue.UI.Components.WebApp/wwwroot/govuk-frontend/assets/js/govuk-frontend.min.js [hunk 4]
```diff
@@ -1 +0,0 @@
...r(".govuk-tabs__list-item--selected .govuk-tabs__tab")},h.prototype.getHref=function(t){var e=t.getAttribute("href");return e.slice(e.indexOf("#"),e.length)},t.initAll=function(t){var o=void 0!==(t=void 0!==t?t:{}).scope?t.scope:document;e(o.querySelectorAll('[data-module="govuk-button"]'),(function(t){new i(t).init()})),e(o.querySelectorAll('[data-module="govuk-accordion"]'),(function(t){new n(t).init()})),e(o.querySelectorAll('[data-module="govuk-details"]'),(function(t){new r(t).init()})),e(o.querySelectorAll('[data-module="govuk-character-count"]'),(function(t){new s(t).init()})),e(o.querySelectorAll('[data-module="govuk-checkboxes"]'),(function(t){new a(t).init()})),new c(o.querySelector('[data-module="govuk-error-summary"]')).init(),new u(o.querySelector('[data-module="govuk-header"]')).init(),e(o.querySelectorAll('[data-module="govuk-notification-banner"]'),(function(t){new l(t).init()})),e(o.querySelectorAll('[data-module="govuk-radios"]'),(function(t){new d(t).init()})),e(o.querySelectorAll('[data-module="govuk-tabs"]'),(function(t){new h(t).init()}))},t.Accordion=n,t.Button=i,t.Details=r,t.CharacterCount=s,t.Checkboxes=a,t.ErrorSummary=c,t.Header=u,t.Radios=d,t.Tabs=h}));
```
Comment (ghost) [inline src/NHSD.GPIT.BuyingCatalogue.UI.Components.WebApp/wwwroot/govuk-frontend/assets/js/govuk-frontend.min.js]: This change can be reverted as I suspect it's unrelated to the tests below

**Review by bjss-colin — CHANGES_REQUESTED** [review]

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/OrderingPages.cs (hunk 1 above)
Comment (samdasari9) [reply tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/OrderingPages.cs]: Made changes as suggested.

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepThree/OrderingStepThree.cs (hunk 2 above)
Comment (samdasari9) [reply tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepThree/OrderingStepThree.cs]: Made changes as suggested.

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepThree/OrderingStepThree.cs (hunk 3 above)
Comment (samdasari9) [reply tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepThree/OrderingStepThree.cs]: Made changes as suggested.

File: src/NHSD.GPIT.BuyingCatalogue.UI.Components.WebApp/wwwroot/govuk-frontend/assets/js/govuk-frontend.min.js (hunk 4 above)
Comment (samdasari9) [reply src/NHSD.GPIT.BuyingCatalogue.UI.Components.WebApp/wwwroot/govuk-frontend/assets/js/govuk-frontend.min.js]: Made changes as suggested.

**Review by bjss-colin — APPROVED** [review]

**Comment by samdasari9** [comment]
Bespoke Milestone changes added

**Pull request merged**